Source/Scripts/cheat_core_player.lua

```diff
@@ -108,147 +87,336 @@ function cheat:teleport_to(line)
+  places["RS_1"] = "x:2464 y:2195 z:150"
+  places["RS_2"] = "x:2668 y:3163 z:136"
+  places["RS_3"] = "x:1669 y:3257 z:53"
+  places["RS_4"] = "x:3615 y:3193 z:172"
+  places["RS_5"] = "x:2454 y:2362 z:100"
+  
+  -- All bandit camp locations for INTERLOPERS Quest.
+  places["IL_1"] = "x:188 y:3266 z:106"
+  places["IL_2"] = "x:536 y:2840 z:87"
+  places["IL_3"] = "x:536 y:2395 z:32"
+  
   -- MOLDAVITE_BANDIT_CAMP
-  places["BANDITS_1"] = "x:365 y:1749 z:19"
-
+  places["BC_1"] = "x:365 y:1749 z:19"
   -- SKALITZ_SMELTERY_BANDIT_CAMP
-  places["BANDITS_2"] = "x:873 y:3279 z:23"
-
-  -- SASSAU_BANDIT_CAMP
-  places["BANDITS_3"] = "x:1295 y:1707 z:40"
-
+  places["BC_2"] = "x:873 y:3279 z:23"
+  -- SASAU_BANDIT_CAMP
+  places["BC_3"] = "x:1295 y:1707 z:40"
   -- MONASTERY_BANDIT_CAMP
-  places["BANDITS_4"] = "x:696 y:2115 z:52"
-  
+  places["BC_4"] = "x:696 y:2115 z:52"
   -- North of Merhojed
-  places["BANDITS_5"] = "x:1731 y:2977 z:132"
-  
+  places["BC_5"] = "x:1731 y:2977 z:132"
   -- east of skalitz
-  places["BANDITS_6"] = "x:1274 y:3499 z:87"
-  
+  places["BC_6"] = "x:1274 y:3499 z:87"
   -- west of Rattai
-  places["BANDITS_7"] = "x:1271 y:532 z:156"
-  
+  places["BC_7"] = "x:1271 y:532 z:156"
   -- north of monastery
-  places["BANDITS_8"] = "x:692 y:2110 z:52"
-  
+  places["BC_8"] = "x:692 y:2110 z:52"
   -- West of vranik (poachers)
-  places["BANDITS_9"] = "x:229 y:999 z:156"
-  
+  places["BC_9"] = "x:229 y:999 z:156"
   -- cumans southeast of talmberg (not raiders? they drop ears)
-  places["BANDITS_10"] = "x:2697 y:2187 z:126"
+  places["BC_10"] = "x:2697 y:2187 z:126"
   
+  if gender ~= 2 then
+    if not nplaceErr then
+      if places[cheat:toUpper(nplace)] ~= nil then
+        cheat:teleport(places[cheat:toUpper(nplace)])
+      else
+        cheat:logError("Invalid Place - See list of supported places type: 'cheat_tp_bc ?'")
+      end
+    end
+  else
+    cheat:logError("You can't use this command while playing Thereza!")
+  end
+end
+
+-- ============================================================================
+-- cheat_tp_bh (BATH HOUSE)
+-- ============================================================================
+cheat.cheat_tp_bh_args = {
+  place = function(args,name,showHelp) return cheat:argsGetRequired(args, name, showHelp, "Teleport to bath house") end,
+}
+
+cheat:createCommand("cheat_tp_bh", "cheat:tp_bh(%line)", cheat.cheat_tp_bh_args,
+"Teleports the player to the given place. Supported places (case insensitive):\n$8LEDETCHKO, RATTAY, SASAU, TALMBERG",
+"Example", "cheat_tp_bh place:SASAU")
+function cheat:tp_bh(line)
+  local gender = player.soul:GetGender()
+  local args = cheat:argsProcess(line, cheat.cheat_tp_bh_args)
+  local nplace, nplaceErr = cheat:argsGet(args, 'place')
+  
+  local places = {}
+  places["LEDETCHKO"] = "x:2042 y:1282 z:28"
+  places["RATTAY"] = "x:2380 y:595 z:31"
+  places["SASAU"] = "x:959 y:1317 z:19"
+  places["TALMBERG"] = "x:2390 y:2810 z:89"  
```

Comment:
Is it worth just numbering the keys now that each location type has its own command?
ex: cheat:tp_bh 1

Reply:
Good point. But what should the description look like then? LEDETCHKO = 1, RATTAY = 2 or LEDETCHKO (1), RATTAY (2) or (LEDETCHKO) 1, (RATTAY) 2?